src/components/MusicPlayer/ThePlayerBar.vue

```diff
@@ -393,6 +393,20 @@ It can be an album/playlist or the original song lists where this song is from
 
   created() {
     window.addEventListener('keydown', this.handleKeyDown);
+
+    //Add MediaKey Support using MediaSession
+    navigator.mediaSession.setActionHandler("nexttrack", _ => {
```

Comment:
Got an ESLint error for "_" as its type is implicit. You can replace "_" with () => ....